site/tutorials/keptn-multistage-qualitygates-08.md

```diff
@@ -0,0 +1,479 @@
+summary: Multistage deployment with Quality Gates using Prometheus
+id: keptn-multistage-qualitygates-08
+categories: Prometheus,aks,eks,gke,openshift,pks,minikube,full-tour,quality-gates,automated-operations
+tags: keptn08x
+status: Published
+authors: Gabriel Tanner
+Feedback Link: https://github.com/keptn/tutorials/tree/master/site/tutorials
+
+# Multistage deployment with Quality Gates using Prometheus
+
+## Welcome
+Duration: 2:00
+
+In this tutorial we'll set up a demo application which will feature different Prometheus metrics and deploy the application using multistage delivery. We will then use Keptn quality gates to evaluate the resilience of the application based on SLO-driven quality gates.
```

Comment:
In this tutorial we'll set up the Podtato-head demo application which will feature different Prometheus metrics and deploy the application using multistage delivery. We will then use Keptn quality gates to evaluate the quality of the application based on SLO-driven quality gates.